indico/web/forms/widgets.py

```diff
@@ -166,10 +166,14 @@ class RemoteDropdownWidget(JinjaWidget):
                         item label.
     :param search_field: The attribute of the response used to search
                          in locally available data.
+    :param choices: The dropdown choices when not fetching dynamically.
+    :param allow_additions: Whether to allow adding new values to the options.
+    :param multiple: Whether to allow selecting multiple values.
     """
 
     def __init__(self, search_url=None, search_method='GET', min_trigger_length=3, allow_by_id=False, preload=False,
```

Comment:
```suggestion
    def __init__(self, *, search_url=None, search_method='GET', min_trigger_length=3, allow_by_id=False, preload=False,
```

just a general improvement - all those args are passed as kwargs anyway, and should never be passed as positional args (even less so now tat there are many more)